.github/actions/checkout-submodules-and-bootstrap/action.yaml

```diff
@@ -34,8 +34,10 @@ runs:
       # this point onwards.
       shell: bash
       run: |
-        du -sh $HOME/.cipd-cache-dir/
-        rm -rf $HOME/.cipd-cache-dir/
+        if [ -d $HOME/.cipd-cache-dir/ ]; then
+          du -sh $HOME/.cipd-cache-dir/
+          rm -rf $HOME/.cipd-cache-dir/
+        fi
```

Comment:
Or you could just do:
```
    rm -rf $HOME/.cipd-cache-dir/ || echo "Removing cipd cache dir failed"
```
or something...  Realistically, if the dir _does_ exist and removing it fails for some reason, we would not want to fail the job for that, right?

Reply:
Thanks! I agree this is the better approach.